script.js

```diff
@@ -0,0 +1,156 @@
+let todoArr = [];
+let doneArr = [];
+
+/*로컬 스토리지에서 가져오기 함수*/
+const getLocalStorage = () => {
+    const localTodo = window.localStorage.getItem('todo');
+    const localDone = window.localStorage.getItem('done');
+    console.log(localTodo, localDone);
+
+    //로컬스토리지에 데이터가 있을때에만 가져온다
+    if (localTodo) {
+        todoArr = JSON.parse(localTodo);
+        todoArr.map((v) => createTodoElement(v, 'todo'));
+    }
+    if (localDone) {
+        doneArr = JSON.parse(localDone);
+        doneArr.map((v) => createTodoElement(v, 'done'));
+    }
+};
+
+//현재 배열을 로컬스토리지로 동기화
+const syncLocalStorage = () => {
+    window.localStorage.clear();
+    window.localStorage.setItem('todo', JSON.stringify(todoArr));
+    window.localStorage.setItem('done', JSON.stringify(doneArr));
+    console.log('sync');
+};
+
+/*아이템 생성하기 함수*/
+const createTodoElement = (content, isItToDo) => {
+    const todoList = document.getElementById('todo-list');
+    const doneList = document.getElementById('done-list');
+
+    const item = document.createElement('li');
+    item.className = 'item ';
+    item.className += isItToDo;
```

Comment:
```suggestion
    item.className = `item ${isItToDo}`;
```
템플릿 리터럴을 사용하면 간략하게 쓸 수 있습니다!